core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -746,50 +746,45 @@ public boolean hasTop() {
     @Override
     public void next() throws IOException {
```

Comment:
Ok -- @milleruntime this is one example of something that would be good to check is still the case. I *assume* some of what Adam tested on his own is still applicable but it would be good.

I think we can see this sort of stuff with special JVM opts, but can't point you to it off the cuff.